lib/auth/auth_with_roles_test.go

```diff
@@ -8454,17 +8454,13 @@ func TestCloudDefaultPasswordless(t *testing.T) {
 
 			// create preexisting users
 			for i := 0; i < tc.qtyPreexistingUsers; i += 1 {
-				_, _, err = CreateUserAndRole(srv.AuthServer, fmt.Sprintf("testuser-%d", i), nil, []types.Rule{
-					types.NewRule(types.KindUser, services.RW()),
-				})
+				_, _, err = CreateUserAndRole(srv.AuthServer, fmt.Sprintf("testuser-%d", i), nil, []types.Rule{})
```

Comment:
You could do:

```go
				_, _, err = CreateUserAndRole(srv.AuthServer, fmt.Sprintf("testuser-%d", i), nil, nil)
```

personally I like to sneak in a few comments on ambiguous types for clarity:

```suggestion
				_, _, err = CreateUserAndRole(srv.AuthServer, fmt.Sprintf("testuser-%d", i), nil /* allowedLogins */, nil /* allowRules */)
```